cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -809,21 +844,20 @@ core::PlanNodePtr SubstraitToVeloxPlanConverter::toVeloxPlan(const ::substrait::
         !readRel.common().has_emit(), "Emit not supported for ValuesNode and TableScanNode related Substrait plans.");
   }
 
-  // Check if the ReadRel specifies an input of stream. If yes, the pre-built
-  // input node will be used as the data source.
-  auto splitInfo = std::make_shared<SplitInfo>();
+  // Check if the ReadRel specifies an input of stream. If yes, build ValueStreamNode as the data source.
   auto streamIdx = getStreamIndex(readRel);
```

Comment:
`getStreamIndex` relies on local files to extract the iterator index. Could you help confirm?

Reply:
yes.